restapi/src/test/java/io/stargate/web/resources/ConvertersTest.java

```diff
@@ -0,0 +1,201 @@
+package io.stargate.web.resources;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import com.datastax.oss.driver.api.core.data.CqlDuration;
+import com.datastax.oss.protocol.internal.util.Bytes;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.stargate.db.schema.Column;
+import io.stargate.db.schema.Column.ColumnType;
+import io.stargate.db.schema.Column.Type;
+import io.stargate.db.schema.ImmutableColumn;
+import io.stargate.db.schema.ImmutableUserDefinedType;
+import io.stargate.db.schema.ParameterizedType.TupleType;
+import io.stargate.db.schema.UserDefinedType;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.InetAddress;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Collections;
+import java.util.UUID;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class ConvertersTest {
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+  private static final TupleType INT_TEXT_TUPLE = (TupleType) Type.Tuple.of(Type.Int, Type.Text);
+  private static final UserDefinedType ADDRESS_UDT =
+      ImmutableUserDefinedType.builder()
+          .keyspace("directory")
+          .name("address")
+          .addColumns(
+              ImmutableColumn.builder()
+                  .name("street")
+                  .type(Type.Text)
+                  .kind(Column.Kind.Regular)
+                  .build(),
+              ImmutableColumn.builder()
+                  .name("zip")
+                  .type(Type.Int)
+                  .kind(Column.Kind.Regular)
+                  .build())
+          .build();
+
+  @ParameterizedTest
+  @MethodSource("toJsonSamples")
+  @DisplayName("Should coerce CQL value to JSON")
+  public void toJsonValueTest(Object cqlValue, String expectedJson) throws JsonProcessingException {
+    String actualJson = OBJECT_MAPPER.writeValueAsString(Converters.toJsonValue(cqlValue));
+    assertThat(actualJson).isEqualTo(expectedJson);
+  }
+
+  private static Arguments[] toJsonSamples() throws Exception {
+    return new Arguments[] {
+      arguments(
+          UUID.fromString("59e86020-3502-11eb-bc1b-53a4f383ccdf"),
+          "\"59e86020-3502-11eb-bc1b-53a4f383ccdf\""),
+      arguments("abc", "\"abc\""),
+      arguments(CqlDuration.newInstance(0, 1, 0), "\"1d\""),
+      arguments(1L, "\"1\""),
+      arguments(BigInteger.ONE, "\"1\""),
+      arguments(BigDecimal.ONE, "\"1\""),
```

Comment:
I'm confused, why are these a string "1" rather than number 1?

Reply:
Is it the same for a boolean?

Reply:
Maybe it's because the ranges of the types long (64-bit integer), bigint and bigdecimal cannot be represented in a json number type?

Reply:
Yes, it's for interoperability issues. JSON itself does not impose limitations on numbers; but REST APIs are often consumed from Javascript clients, which don't properly handle integers above 2^53. This can be observed in your browser's Javascript console: 
```
> JSON.parse("9223372036854775807")
9223372036854776000
```

This only affects Long, BigInteger and BigDecimal. Boolean and the smaller int types are returned as their direct JSON counterparts. I forgot to add test samples for them, will do.

Reply:
Also forgot to mention that for BigDecimal it's not only a matter of range, but also precision. The same reason you don't use `float` or `double` to represent exact values in pure Java.